### ISSUE TITLE
Update labels: Issue Template: blank-issue-form-with-no-dependency

### ISSUE THREAD

**Issue by steven-positive-tran** [hackforla/website#4485]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] Navigate to `.github/ISSUE_TEMPLATE/blank-issue-form-with-no-dependency.yml`
- [x] Add the following line under the description field
   ```
   labels: ['role missing', 'Complexity: Missing', 'Feature Missing', 'size: missing','Draft']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template
URL of the issue branch on the test Repository:  https://github.com/vincentdang/website/issues/new?assignees=&labels=role+missing%2CComplexity%3A+Missing%2CFeature+Missing%2Csize%3A+missing%2CDraft&projects=&template=blank-issue-form-with-no-dependency.yml

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/blank-issue-form-with-no-dependency.yml`
- This issue is part of epic #4307

**Comment by roslynwythe** [comment]
Hi @rdhmdhl please review procedures for completing issues and creating Pull Requests in Part 2 and Part 3 of our [CONTRIBUTING document](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues).  Developers are not supposed to close their own issues, rather issues are closed when the linked Pull Request in merged.  I am re-opening this issue.

**Comment by roslynwythe** [comment]
@rdhmdhl This issue is in the "New Issue Approval" column and has not been approved or prioritized, so I will be removing your assignment.   I also added a final Action Item regarding the test procedure for this issue, however those instructions are not ready yet.  Please note that developers should only take on issues from the "Prioritized Backlog" column.  Thank you.

**Comment by vincentdang** [comment]
i. Availability: M-F after 6-7pm
ii. ETA: within a week or two.

**Comment by vincentdang** [comment]
Update: I was able to create a new branch, add the labels, and push it upstream.  However I realize my branch was a bit outdated as it is many commits behind. Is this a problem or can I begin testing?  @steven-positive-tran  
<img width="737" alt="image" src="https://github.com/hackforla/website/assets/1876652/e8e66f34-5e83-445a-8e04-76b26d42baaf">
<img width="1279" alt="image" src="https://github.com/hackforla/website/assets/1876652/53b66a8f-e241-4bc6-9f5a-58cfd7ed06f3">

**Comment by steven-positive-tran** [comment]
I'm unfamiliar with this issue since I created it in a meeting but have you tried ```git pull``` to pull all the changes from the hackforla repo to your own branch so its up to date.

**Comment by vincentdang** [comment]
@steven-positive-tran how would I delete the update-addLabel branch?
also my upstream -website branch is 8 commits ahead of hfla/website
They are full of changes that were from past issues.



<img width="984" alt="image" src="https://github.com/hackforla/website/assets/1876652/ddd81041-6f64-43db-ac6f-c3a761847080">

**Comment by vincentdang** [comment]
Update: I have made a pull request.
@steven-positive-tran

**Comment by vincentdang** [comment]
Update: I deleted the branch I accidentally forked and published from the upstream hackforla repo and created a branch and published it in the origin (my repo) with the updates.  I accidentally merged the updated branch to my origin branch via pull request and undid it.  Sorry haha.

**Comment by vincentdang** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
(https://github.com/vincentdang/website/pull/1)
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image]<img width="1278" alt="image" src="https://github.com/hackforla/website/assets/1876652/ac06e186-8ac6-420d-b42b-cf5f1304d700">

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image]
<img width="985" alt="image" src="https://github.com/hackforla/website/assets/1876652/b71c3fbc-760e-46f7-8ef1-4a7817ce2701">


</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image]<img width="1273" alt="image" src="https://github.com/hackforla/website/assets/1876652/84e3d9e3-4fac-4bf0-a92e-da0e23704f0d">

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: REPLACE THIS TEXT WITH URL OF THE ISSUE TEMPLATE ON THE TEST REPO
 This will enable others to check the template.  
- [x] Verify the appearance of the issue template, including careful examination of the labels.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [ ] Complete the PR process by visiting the URL saved from the push command
- [x] Copy these sections from the Issue into the PR 
  - [x] "For PR Reviewers and Merge Team"
  - [x] "Link for Reviewers"
  - [x] "For PM, Merge Team, or Tech Lead"

**Comment by vincentdang** [comment]
### Instructions for testing updated issue templates:
- [ ] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [ ] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
REPLACE THIS TEXT WITH URL FOR CREATING PR
- [ ] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [ ] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [ ] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [ ] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [ ] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [ ] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [ ] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: REPLACE THIS TEXT WITH URL OF THE ISSUE TEMPLATE ON THE TEST REPO
 This will enable others to check the template.  
- [ ] Verify the appearance of the issue template, including careful examination of the labels.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [ ] Complete the PR process by visiting the URL saved from the push command
- [ ] Copy these sections from the Issue into the PR 
  - [ ] "For PR Reviewers and Merge Team"
  - [ ] "Link for Reviewers"
  - [ ] "For PM, Merge Team, or Tech Lead"

**Comment by vincentdang** [comment]
@roslynwythe in the comment above, where is the resources section of the issue where I replace the text with the url? https://github.com/vincentdang/website/issues/new?assignees=&labels=role+missing%2CComplexity%3A+Missing%2CFeature+Missing%2Csize%3A+missing%2CDraft&projects=&template=blank-issue-form-with-no-dependency.yml

**Comment by LRenDO** [comment]
> @roslynwythe in the comment above, where is the resources section of the issue where I replace the text with the url? https://github.com/vincentdang/website/issues/new?assignees=&labels=role+missing%2CComplexity%3A+Missing%2CFeature+Missing%2Csize%3A+missing%2CDraft&projects=&template=blank-issue-form-with-no-dependency.yml

Hi @vincentdang! It looks like the resource heading might not have been included in this issue.  The link should be added in the `Link for Reviewers` section of this issue by replacing the text `REPLACE THIS TEXT AND BRACKETS WITH THE URL`.  Please also see the requests for changes on the pull request for this issue #4953.  Feel free to ping me if you have any questions!

**Comment by vincentdang** [comment]
Progress: Copied and pasted the url into pull request as well as PR/Merge team checkboxes and issue number and link.
Blockers: Need to find url in push command.
Availability: This weekend and next week from W-F during evenings.
ETA: By next week.

**Comment by vincentdang** [comment]
> > @roslynwythe in the comment above, where is the resources section of the issue where I replace the text with the url? https://github.com/vincentdang/website/issues/new?assignees=&labels=role+missing%2CComplexity%3A+Missing%2CFeature+Missing%2Csize%3A+missing%2CDraft&projects=&template=blank-issue-form-with-no-dependency.yml
> 
> Hi @vincentdang! It looks like the resource heading might not have been included in this issue. The link should be added in the `Link for Reviewers` section of this issue by replacing the text `REPLACE THIS TEXT AND BRACKETS WITH THE URL`. Please also see the requests for changes on the pull request for this issue #4953. Feel free to ping me if you have any questions!

DONE.

**Comment by vincentdang** [comment]
> > @roslynwythe in the comment above, where is the resources section of the issue where I replace the text with the url? https://github.com/vincentdang/website/issues/new?assignees=&labels=role+missing%2CComplexity%3A+Missing%2CFeature+Missing%2Csize%3A+missing%2CDraft&projects=&template=blank-issue-form-with-no-dependency.yml
> 
> Hi @vincentdang! It looks like the resource heading might not have been included in this issue. The link should be added in the `Link for Reviewers` section of this issue by replacing the text `REPLACE THIS TEXT AND BRACKETS WITH THE URL`. Please also see the requests for changes on the pull request for this issue #4953. Feel free to ping me if you have any questions!

where is the url from the push command?

**Comment by vincentdang** [comment]
@LRenDO hi. am I done?

**Comment by LRenDO** [comment]
Hi @vincentdang! The update to the issue with the link looks good here. I am going to go check out the pull request and leave any comments related to the PR itself and what else may need to be done over there.

**Comment by vincentdang** [comment]
@LRenDO okay let me know if you need me to do anything else.

**Comment by LRenDO** [comment]
> @LRenDO okay let me know if you need me to do anything else.

Hi @vincentdang! It looks like there may still be an issue with branches. Please see my comments on [pull request 4953](https://github.com/hackforla/website/pull/4953#issuecomment-1660870524).

**Comment by LRenDO** [comment]
This was fixed and merged in #5162. Looks like the "Fixes" was missing from the issue linking at the top. Closing now.
